metaschema-cli/src/main/java/gov/nist/secauto/metaschema/cli/commands/AbstractValidateContentCommand.java

```diff
@@ -221,7 +221,7 @@ public ExitStatus execute() {
           String toFormatText = cmdLine.getOptionValue(AS_OPTION);
           asFormat = Format.valueOf(toFormatText.toUpperCase(Locale.ROOT));
         } catch (IllegalArgumentException ex) {
-          return ExitCode.FAIL
+          return ExitCode.INPUT_ERROR
```

Comment:
Minor, non-blocking. If you like the above change:

```suggestion
          return ExitCode.IO_ERROR
```